src/test/test-copy.c

```diff
@@ -1,5 +1,7 @@
 /* SPDX-License-Identifier: LGPL-2.1-or-later */
 
+#include <openssl/ossl_typ.h>
+#include <string.h>
```

Comment:
Do we need this include? I'm pretty sure it'll all still compile without adding this